dashdotdb/schemas/swagger.yaml

```diff
@@ -11,38 +11,55 @@ info:
 servers:
 - url: /api/v1
 paths:
-  /imagemanifestvuln/metrics:
+  # Tokens
+  /token:
     get:
-      operationId: dashdotdb.controllers.metrics.imagemanifestvuln.search
-      summary: Lists Image Manifests Vulnerabilities
+      summary: Create a token for data upload
+      parameters:
+      - name: scope
+        in: query
+        required: true
+        schema:
+          type: string
       responses:
         "200":
-          description: successful operation
+          description: token created
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/ImageManifestVulnReport'
-                x-content-type: application/json
-  /deploymentvalidation/metrics:
-    get:
-      operationId: dashdotdb.controllers.metrics.deploymentvalidation.search
-      summary: Lists Deployment Validation Information
+                type: object
+                properties:
+                  token:
+                    type: string
+                    description: A unique token identifier
+        "405":
+          description: invalid scope
+    post:
```

Comment:
isn't this goin to be a DELETE?

Reply:
Keeping as a POST simply because it's already received a lot of testing and other operations seem to have their own issues.

Reply:
You're not submitting data... the idea of POSTing in this endpoint is very strange and its effect not intuitive at all. DELETE at least conveys the idea of this token not being usable anymore which is what we want

Reply:
True, but a client could have asked for more than 1 token to upload more than 1 type of data and gotten itself confused.  Our integrations don't do that now, but trying not to think about how we currently use the api.

Reply:
for every token that a client asks for, it has to release it. Whenever it releases it, it doesn't submit any data, it just hits the `/token/<token-id>?scope=<scope>` url.